content/references/xrp-ledger-toml.md

```diff
@@ -169,6 +169,7 @@ For _each_ `[[VALIDATORS]]` entry, you MAY provide any of the following fields:
 | Field        | Type   | Description                                          |
 |:-------------|:-------|:-----------------------------------------------------|
 | `public_key` | String | The master public key of your primary validator, encoded in the XRP Ledger's base58 format (typically, this starts with `n`). |
+| `attestation`| String | The signed attestation message that is output by the `set_domain` command in the [validator-keys-tool](https://github.com/ripple/validator-keys-tool)
```

Comment:
Rather than linking the validator keys tool here maybe it would make more sense to link the domain verification steps in this same page? Also, I think it would make more sense to focus on what the attestation is for, not what tool provides it.

Suggested revised description:

> A signed message, in hexadecimal, indicating that the same entity runs this validator and the domain serving this TOML file. For more information, see [Domain Verification](#domain-verification).

Reply:
Thanks! I made the change.